src/emqtt_bench.erl

```diff
@@ -557,18 +577,11 @@ inc_counter(CntName, Inc) ->
 cnt_ref() -> persistent_term:get(?MODULE).
 
 run(Parent, PubSub, Opts, AddrList, HostList) ->
-    run(Parent, proplists:get_value(count, Opts), PubSub, Opts, AddrList, HostList).
-
+    ok = run(Parent, 0, proplists:get_value(count, Opts), PubSub, Opts, AddrList, HostList).
```

Comment:
Nit: I think this will make `run/5` no longer tail recursive.

Reply:
good point, will not fix though, because it's just one stack push.